docs/ConvertDataOperation.md

```diff
@@ -0,0 +1,110 @@
+        },
+        {
+            "name": "rootTemplate",
+            "valueString": "ADT_A01"
+        }
+    ]
+}
+```
+
+**Sample response:**
+
+Note the Content-Type is the response header is set to `text/plain` because the output format is determined by the mapping definition from your templates.
+
+```
+{
+  "resourceType": "Bundle",
+  "type": "transaction",
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:9d697ec3-48c3-3e17-db6a-29a1765e22c6",
+      "resource": {
+        "resourceType": "Patient",
+        "id": "9d697ec3-48c3-3e17-db6a-29a1765e22c6",
+        ...
+        ...
+      "request": {
+        "method": "PUT",
+        "url": "Location/50becdb5-ff56-56c6-40a1-6d554dca80f0"
+      }
+    }
+  ]
+}
+```
+
+### 3. Evaluate default templates
+When you deploy the FHIR server, a copy of the latest template released by the [FHIR Converter](https://github.com/microsoft/FHIR-Converter/tree/dotliquid) project is stored with the FHIR server. You can use these default templates by passing ```microsofthealth/fhirconverter:default``` in the request payload as described earlier.
+
+If the data conversion using default templates meets your requirements, you are all set. Otherwise, you should customize the templates as described below.
+
+### 4. Customize templates
+Use the [FHIR Converter](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-health-fhir-converter) Visual Studio Code extension to customize the templates as per your need. You will first need to install [Visual Studio Code](https://code.visualstudio.com/) if do not have it already.
+
+### 5. Publish your templates to Azure Container Registry (ACR)
+The FHIR server can read custom templates from the [ACR](https://azure.microsoft.com/en-us/services/container-registry/). Create a container registry, and use the [Template Management CLI](https://github.com/microsoft/FHIR-Converter/blob/dotliquid/docs/TemplateManagementCLI.md) tool to push the customized templates to the ACR.
+
+### 6. Make templates accessible to the FHIR server
+
+There are two steps needed to make the templates accessible to the FHIR server at run time.
+
+a) Provide `AcrPull` permission to your FHIR service on the ACR you created.
+
+![acr-rbac](./images/convert-data/acr-rbac.png)&nbsp;
+
+
+b) Register the ACR on your FHIR server by setting `FhirServer:Operations:ConvertData:ContainerRegistryServers:0` to your ACR login server. You can register multiple ACRs by adding more entries in the configuration as the following picture shows.  
+
+![acr-registration](./images/convert-data/acr-registration.png)&nbsp;
+
+### 7. Verify
+
+Make a call to the $convert-data API specifying your template reference in the templateCollectionReference parameter. You can use any one of the following two formats to specify the reference. However, we strongly recommend choosing digest because its immutable and stable.
+
+ `<RegistryServer>/<imageName>:<imageTag>` 
+
```

Comment:
super nit: If we suggest digest I would put it first in a list.

Reply:
same statement for table in  ### 2. Make API call

Reply:
Fixed. Thank you